src/controllers/authenticationController.ts

```diff
@@ -215,6 +218,13 @@ export default class AuthenticationController {
       : E.right(O.none);
 
     if (E.isLeft(errorOrMaybeAssertionRef)) {
+      this.appInsightsTelemetryClient?.trackEvent({
```

Comment:
Add test

Reply:
done in https://github.com/pagopa/io-backend/pull/996/commits/275e94a174fed76bf20736cb6040e924ad01a415